themes/bootstrap3/templates/RecordDriver/SolrOverdrive/previewlink.phtml

```diff
@@ -0,0 +1,5 @@
+<?php
+$previews = $this->driver->getPreviewLinks();
+if ($previews->url): ?>
+    <a href="<?=$previews->url?>>" target="_blank" class="btn btn-info btn-xs"><?=$this->transEsc('Preview')?> </a>
```

Comment:
Should there be a `$this->escapeHtmlAttr()` around `$previews->url`? Is the blank space really wanted before the ` </a>`?